spec/index.html

```diff
@@ -443,32 +520,68 @@ <h3>Triples</h3>
 
     <ul>
       <li>the <dfn>subject</dfn>, which is an
-      <a>IRI</a> or a <a>blank node</a></li>
+      <a>IRI</a>, a <a>blank node</a>, or a <a>quoted triple</a></li>
 
       <li>the <dfn>predicate</dfn>, which is an <a>IRI</a></li>
 
       <li>the <dfn>object</dfn>, which is an <a>IRI</a>,
-      a <a>literal</a> or a <a>blank node</a></li>
+      a <a>literal</a>, a <a>blank node</a>, or a <a>quoted triple</a></li>
     </ul>
 
     <p>An RDF triple is conventionally written in the order subject,
       predicate, object.</p>
 
+    <p>A <dfn data-local-lt="quoted">quoted triple</dfn> is an <a>RDF triple</a>
+      used as the <a>subject</a> or <a>object</a> of another triple.</p>
+
+    <p>An <dfn data-local-lt="asserted">asserted triple</dfn>
+      is an <a>RDF triple</a> that is an element of an <a>RDF graph</a>.</p>
+
+    <p>A <dfn>complex triple</dfn> is an <a>RDF triple</a>
+      having a <a>quoted triple</a> as a component.</p>
+
+    <p>A <dfn>simple triple</dfn> is an <a>RDF triple</a>
+      that does not have a <a>quoted triple</a> as a component.</p>
+
+    <p>A <dfn class="export">complex graph</dfn> is
+      an <a>RDF graph</a> containing one or more <a>complex triples</a>.</p>
+
+    <p>A <dfn class="export">simple graph</dfn> is
+      an <a>RDF graph</a> composed entirely of <a>simple triples</a>.</p>
+
+    <p>The <dfn data-lt="constituent">constituent terms</dfn> (or simply <a>constituents</a>)
+      of a <a>triple</a>
+      is the set containing its <a>subject</a>, its <a>predicate</a>, its <a>object</a>,
+      plus all the <a>constituent terms</a> of its <a>subject</a> and/or its <a>object</a>
+      if they are themselves <a>quoted triples</a>.
+      By extension, we define the <a>constituent terms</a> of a <a>complex graph</a>
```

Comment:
Do we need this extension in RDF Concept in the primary section about triple?

If used in the semantics, put the referable terminology there.

Reply:
I think we should recognize "constituent terms" and "nodes". We may not need "constituent terms" either of a triple or a graph, at all, but it was in the CG report. But, the term "nodes" would by that same logic to better be in semantics than concepts.